app/server/src/services/auth.js

```diff
@@ -0,0 +1,100 @@
+//We will do the authorization checks here !!
+const jwt = require('jsonwebtoken');
+const UserModel = require("../models/user/user.model");
+const crypto = require("crypto");
+const jwt_ac_secret = process.env.JWT_AC_KEY
```

Comment:
I see that these env variables are not added to .env.example file... Let's add them there

Reply:
Done!